spec/helpers/web_helpers.rb

```diff
@@ -0,0 +1,16 @@
+def add_listing(day: "Monday")
+  click_link "Add a class"
+  fill_in "listing_day", with: day
+  select 10, from: "listing_hour"
+  select 0, from: "listing_mins"
+  click_button "Create Listing"
+end
+
+def sign_up (email: "test@example.com", name: "Tippee Toes", password: "password")
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)